coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
       href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext'>
     <link rel="stylesheet"
-          href="bower_components/bootstrap/dist/css/bootstrap.css">
+    href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```

Comment:
You might want to use something like `http://getbootstrap.com/dist/css/bootstrap.css` that will automatically update version (thereby get bug fixes without any work from our part), maybe?